pages/conference/_eventType/_id.vue

```diff
@@ -0,0 +1,329 @@
+<template>
+    <i18n-page-wrapper class="px-8 sm:px-10 md:px-32 lg:px-60" custom-x>
+        <core-h1 :title="data.title" center class="font-black"></core-h1>
+        <div class="speech__speakers">
+            <div
+                v-for="(speaker, index) in data.speakers"
+                :key="`speech_details_speaker_${index}`"
+                class="speech_speakerContainer"
+            >
+                <div class="speech__speakerThumbnail">
+                    <img :src="speaker.thumbnail_url" :alt="speaker.name" />
+                </div>
+                <p class="speech__speakerName">{{ speaker.name }}</p>
+            </div>
+        </div>
+
+        <div class="speech__infoSection">
+            <div class="speech__infos">
+                <div class="speech__info">
+                    <img :src="icons.location" class="mx-2" />
+                    {{ locationMapping[data.location] }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.language" class="mx-2" />
+                    {{ $t(`languages.${data.language}`) }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.datetime" class="mx-2" />
+                    {{ $t(data.dateTag) }} • {{ getTime(data.begin_time) }}-{{
+                        getTime(data.end_time)
+                    }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.level" class="mx-2" />
+                    {{ $t(`levels.${data.python_level}`) }}
+                </div>
+                <div class="speech__info">
+                    <img :src="icons.category" class="mx-2" />
+                    {{ $t(`categories.${data.category}`) }}
```

Comment:
1. Give more spaces between the icon and text to align with the design spec better so that it feels more relaxing:

| Actual | Expected |
|:--------|:----------|
|<img width="400" alt="Screen Shot 2021-08-06 at 6 16 44 AM" src="https://user-images.githubusercontent.com/7352613/128428469-820f66c2-3952-431e-bd42-338695694cf0.png">|<img width="400" alt="Screen Shot 2021-08-06 at 6 18 09 AM" src="https://user-images.githubusercontent.com/7352613/128428616-ed2d3601-d145-42f7-89fb-18e022fb4075.png">|
2. According to the above Actual image, use a bloder font-weight to match the design
3. Neet to provide `alt=""` for all `img`

Reply:
modified
![image](https://user-images.githubusercontent.com/24987826/128453766-9d57dd19-5af7-448c-9b36-38ebeacf7b9b.png)